docs/about/concepts/video/abstractions.md

```diff
@@ -58,12 +58,6 @@ Composite stages provide a user-facing convenience API and decompose into one or
 
 ```python
 class MyStage(ProcessingStage[X, Y]):
-    @property
-    def name(self) -> str: ...
-
-    @property
-    def resources(self) -> Resources: ...
-
     def inputs(self) -> tuple[list[str], list[str]]: ...
```

Comment:
Should we still tell them how to use the name and resources? Like as a variable instead of func?